.github/workflows/integration_test.yaml

```
@@ -81,7 +68,13 @@ jobs:
           " >> /var/snap/microk8s/current/args/containerd-template.toml'
           sudo su -c 'systemctl restart snap.microk8s.daemon-containerd.service && microk8s status --wait-ready'
       - name: Run integration tests
-        run: tox -e integration -- --model testing --indico-image ${{ env.REGISTRY }}/${{ env.OWNER }}/${{ env.INDICO_IMAGE_NAME }}:${{ github.run_id }} --indico-nginx-image ${{ env.REGISTRY }}/${{ env.OWNER }}/${{ env.INDICO_NGINX_IMAGE_NAME }}:${{ github.run_id }}
+        run: |
+          image_names=$(echo '${{ needs.get-images.outputs.images }}' | jq -c '.[]' | sed 's/\"//g')
```

You can use "jq -cr" and the output will be without double-quote, so you can remove the sed
```suggestion
          image_names=$(echo '${{ needs.get-images.outputs.images }}' | jq -cr '.[]')
```